install-steps.md

```diff
@@ -1,193 +1,821 @@
+    - name: worker
+      replicas: 2
+    controlPlane:
+      name: master
+      replicas: 3
+      platform:
+        baremetal: {}
+    platform:
+      baremetal:
+        apiVIP: <api-ip>
+        ingressVIP: <wildcard-ip>
+        dnsVIP: <dns-ip>
+        provisioningBridge: provisioning
+        externalBridge: baremetal
+        hosts:
+          - name: openshift-master-0
+            role: master
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address>
+            hardwareProfile: default
+          - name: openshift-master-1
+            role: master
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address>
+            hardwareProfile: default
+          - name: openshift-master-2
+            role: master
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address
+            hardwareProfile: default
+          - name: openshift-worker-0
+            role: worker
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address
+            hardwareProfile: default
+          - name: openshift-worker-1
+            role: worker
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address
+            hardwareProfile: default
+    pullSecret: '<pull_secret>'
+    sshKey: '<ssh_pub_key>'
+    ~~~
+
+NOTE: Ensure to change the appropriate variables to match your environment.
+
+2. Create a directory to store cluster configs
+    ~~~sh
+    mkdir ~/clusterconfigs
+    cp install-config.yaml ~/clusterconfigs
+    ~~~
+
+3. Ensure all baremetal nodes are powered off prior to installing the OpenShift cluster
+   ~~~sh
+   ipmitool -I lanplus -U <user> -P <password> -H <management-server-ip> -p 6230 power off
+   ~~~
+
+4. Run the OpenShift Installer
+    ~~~sh
+    ./openshift-baremetal-install --dir ~/clusterconfigs --log-level debug create cluster
+    ~~~
+
+## Fix Metal3 ConfigMap
+
+With the installation complete. The focus shifts to fixing the metal3 ConfigMap
+in order to deploy new nodes. 
+
+Due to this [issue](https://github.com/openshift/installer/pull/2449) we need 
+to fix the ConfigMap for the Metal3 operator.
+
+This ConfigMap is used to notify `ironic` how to PXE boot new nodes.
+
+1. Create the sample ConfigMap `metal3-config.yaml.sample`
+   ~~~yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: metal3-config
+      namespace: openshift-machine-api
+    data:
+      cache_url: ${RHCOS_URI}
```

Comment:
I think this is better to set it to `''` to jump directly to download the rhcos file

Reply:
fixed.